src/devtools/views/Components/Tree.js

```diff
@@ -333,23 +333,33 @@ function InnerElementType({ style, ...rest }) {
   // What we can do instead, is passively measure the width of the current rows,
   // and ensure that once we've grown to a new max size, we don't shrink below it.
   // This improves the user experience when scrolling between wide and narrow rows.
-  // We shouldn't retain this width across different conceptual trees though,
-  // so when the user opens the "owners tree" view, we should discard the previous width.
   const divRef = useRef<HTMLDivElement | null>(null);
-  const minWidthRef = useRef<number | null>(null);
-  const minWidth =
-    ownerStack.length > 0 || minWidthRef.current === null
-      ? '100%'
-      : minWidthRef.current;
+  const [minWidth, setMinWidth] = useState(null);
   useEffect(() => {
```

Comment:
This one's failing lint now.

```
src/devtools/views/Components/Tree.js
  339:3  error  React Hook useEffect contains a call to 'setMinWidth'. Without a list of dependencies, this can lead to an infinite chain of updates. To fix this, pass [] as a second argument to the useEffect Hook  react-hooks/exhaustive-deps
```

Reply:
Let's disable this line for now. The warning is fair but I don't know of a better solution yet.

Reply:
K ~ https://github.com/bvaughn/react-devtools-experimental/issues/156